tests/test_model_methods.py

```diff
@@ -121,6 +123,8 @@ async def test_save_partial(self):
         n_mdl = await self.cls.get(id=self.mdl.id)
         self.assertEqual(n_mdl.name, "Test")
         self.assertEqual(n_mdl.desc, "Something")
+        with pytest.raises(OperationalError):
```

Comment:
I would suggest to have a separate test case for this os it is documented with tests and it is not just a check in an existing test. 

I would also suggest another test case:

```
o.save()
o.id = 123
with pytest.raises(OperationalError):
  o.save()
```

Reply:
I tried, but it's hard to do that -- because can't get the original id if it changed which make it difficult to know whether object only changed pk field value.

Reply:
@henadzit Cloud your review again?

Reply:
Sorry, I was traveling and was quite busy recently, so wasn't able to participate much. I'll have a look in coming days, thanks!